File: ios/PortalModels.swift
import Foundation

// MARK: - Domain models (non-API contract)

struct TimelineEvent: Codable, Identifiable {
    let id: String
    let type: TimelineEventType?
    let at: Date?
    let actorName: String?
    let kilnName: String?
    let notes: String?
}

File: android/app/src/main/java/com/monsoonfire/portal/reference/PortalModels.kt
package com.monsoonfire.portal.reference

import kotlinx.serialization.Serializable

// Domain-only models (non-API contract)

@Serializable
data class TimelineEvent(
    val id: String,
    val type: String? = null,
    val at: String? = null,
    val actorName: String? = null,
    val kilnName: String? = null,
    val notes: String? = null
)
